README.md

```diff
@@ -1,13 +1,23 @@
-# Happy Thoughts
+# Happy Thoughts :sparkling_heart:
 
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+This project was to make a kind of Twitter App in React by fetching and posting to an API. The user can post and like happy thoughts.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+### Tech I've used / What I've practiced
+- React.js / State and Effect hooks
+- JavaScript
+- JSX
+- API
+
+The goal was to practice using React state and effect hooks while fetching and posting data to an API. 
+I started with the list of messages, fetching from the API and mapping through the data to create the list. 
+The data is sorted and messages with no input are filtered out.
+I continued to create the input form in which the user can post their happy thoughts. The submit button is disabled if the user puts in too many/few characters and there is a counter to show the user how many characters they have used.
+Finally I added a counter for how many individual posts the user has liked. I did this using a state hook with an array that stores the id of the liked post.
```

Comment:
Great read-me Anna, I really like that you have in depth explained all of your features and your emojis! 😍